examples/experiment_eval.py

```diff
@@ -108,6 +108,20 @@ def _iterate_env_agent_combo_not_in_db(
     env_ids: list[str] = [],
     tag: str | None = None,
 ) -> Generator[EnvAgentCombo[Observation, AgentAction], None, None]:
+    # method 1 for loading evaluation metric
+    evaluation_dimensions = (
```

Comment:
It is not a good thing to write things like this? (remove method 1?) move it as an example to the doc?